plugins/chat/assets/javascripts/discourse/components/chat-channel-unread-indicator.gjs

```diff
@@ -34,12 +30,19 @@ export default class ChatChannelUnreadIndicator extends Component {
       return this.#hasChannelMentions();
     }
     return (
-      this.args.channel.isDirectMessageChannel ||
+      this.#isDirectMessage() ||
```

Comment:
We also want to ensure a regular message is received when it's a direct message, this prevents showing urgent status for unread tracked threads.